src/refmt_impl.ml

```diff
@@ -167,12 +191,14 @@ let () =
         )
         | Some "binary"
         | None -> fun (ast, comments) -> (
-          output_string stdout Config.ast_intf_magic_number;
-          output_value  stdout filename;
-          output_value  stdout ast
+            let (module V) = !ocaml_version in
```

Comment:
Yes. The package exposes modules `OCaml_402`, `OCaml_403`, ... and an alias `OCaml_current`.
When targetting a fixed version of OCaml, say 4.04, `open OCaml_404` shadows (some of) compiler-libs modules with the versioned ones.
When communicating with the rest of the world, a `Convert` functor will transform  between any two versions of the compiler. So `Convert(OCaml_404)(OCaml_current)` is the identity on 4.04 and the appropriate conversion function on other compilers.